extide/gradle/src/org/netbeans/modules/gradle/customizer/GradleExecutionPanel.java

```diff
@@ -110,17 +150,27 @@ private void initComponents() {
     }// </editor-fold>//GEN-END:initComponents
 
     // Variables declaration - do not modify//GEN-BEGIN:variables
-    private javax.swing.JCheckBox cbTrustProject;
+    private javax.swing.JComboBox<String> cbTrustLevel;
     private javax.swing.JLabel lbReadOnly;
+    private javax.swing.JLabel lbTrustLevel;
     private javax.swing.JLabel lbTrustTerms;
     // End of variables declaration//GEN-END:variables
 
     void save() {
         if (project != null) {
-            if (cbTrustProject.isSelected()) {
-                ProjectTrust.getDefault().trustProject(project);
-            } else {
+            Object v = cbTrustLevel.getSelectedItem();
+            if (v == null) {
+                v = LEVEL_NONE;
+            }
+            if (LEVEL_NONE.equals(v)) {
```

Comment:
You have enums, use a switch (It would have worked with strings as well)

Reply:
Fixed in 7e035b7